Runtime/Components/Cinemachine3rdPersonFollow.cs

```diff
@@ -206,7 +206,6 @@ void PositionCamera(ref CameraState curState, float deltaTime)
 
             // Set state
             curState.RawPosition = camPos;
-            curState.RawOrientation = targetRot;
```

Comment:
I'm not so sure this is a good idea anymore.  It will break scenes that have only the 3rdPersonFollow and no 3rdPersonAim.  I think we're stuck leaving this here.  People can still add something in Aim to override this.

Reply:
Ok, but 3rdPersonAim should not rely on curState.RawOrientation. Or should I change that code back?